lib/src/widgets/palette_picker.dart

```diff
@@ -11,9 +11,14 @@ class PalettePicker extends StatefulWidget {
     this.rightPosition = 1.0,
     this.topPosition = 0.0,
     this.bottomPosition = 1.0,
+    this.border,
+    this.borderRadius = const BorderRadius.all(
+      Radius.circular(6),
```

Comment:
Would the formatting still be ok without this trailing comma?

Reply:
fixed